x-pack/platform/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/schedule.ts

```diff
@@ -12,7 +12,7 @@ import { ALERTING_CASES_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-serve
 import type { AdHocRunSO } from '@kbn/alerting-plugin/server/data/ad_hoc_run/types';
 import { get } from 'lodash';
 import { AD_HOC_RUN_SAVED_OBJECT_TYPE } from '@kbn/alerting-plugin/server/saved_objects';
-import { asyncForEach } from '../../../../../../functional/services/transform/api';
+import { asyncForEach } from '../../../../../../api_integration/services/transform/api';
```

Comment:
The plan is to keep api services under api_integration directory, updating import here